File: prog0604/additional_file/main.py
def count_pos_neg(arr: list[int]) -> list[int]
    pass

if __name__ == "__main__":
    n = int(input())
    arr = list(map(int, input().split()))

    res = count_pos_neg(arr)
    print(f"{res[0]} {res[1]}")

File: prog0604/additional_file/main.cpp
#include <bits/stdc++.h>
using namespace std;

void count_pos_neg(int arr[], int n, int res[2]) {
    // your code here
}

int main() {
    int n;
    cin >> n; 

    int arr[10000];
    for(int i = 0; i < n; i++){
        cin >> arr[i];
    }

    int res[2];
    count_pos_neg(arr, n, res);
    cout << res[0] << " " << res[1] << endl;
}
